src/Hedgehog/Range.fs

```diff
@@ -212,8 +222,9 @@ module Range =
     /// >>> Range.bounds 99 <| Range.linear 0 10
     /// (0, 10)
     ///
-    let inline linear (x : 'a) : ('a -> Range<'a>) =
-      linearFrom x x
+    [<CompiledName("Linear")>]
+    let inline linear (x : 'a) (y : 'a) : Range<'a> =
+      linearFrom x x y
```

Comment:
This is so we don't end up with `FSharpFunc` in the API, otherwise it's tricky for C# to call.